ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.17__amd64-nightly-4.17-upgrade-from-stable-4.16.yaml

```diff
@@ -204,28 +225,46 @@ tests:
     test:
     - chain: openshift-upgrade-qe-test-hypershift
     workflow: cucushift-installer-rehearse-aws-ipi-ovn-hypershift-guest
-- as: aws-ipi-shared-vpc-phz-freeipa-f28
-  cron: 40 1 14 * *
+- as: aws-ipi-private-shared-vpc-phz-sts-f28
```

Comment:
Missing `fips` ?

Reply:
in `openshift-openshift-tests-private-release-4.16__amd64-nightly.yaml` it is also without `fip`: aws-ipi-private-shared-vpc-phz-sts-f2